src/dev/i18n/extract_default_translations.js

```diff
@@ -81,33 +81,31 @@ export async function matchEntriesWithExctractors(inputPath, options = {}) {
     '**/*.d.ts',
   ].concat(additionalIgnore);
 
-  const entries = await globAsync('*.{js,jsx,ts,tsx,html}', {
+  const entries = await globAsync('*.{js,jsx,ts,tsx}', {
     cwd: inputPath,
     matchBase: true,
     ignore,
     mark,
     absolute,
   });
 
-  const { htmlEntries, codeEntries } = entries.reduce(
+  const { codeEntries } = entries.reduce(
     (paths, entry) => {
       const resolvedPath = path.resolve(inputPath, entry);
 
-      if (resolvedPath.endsWith('.html')) {
-        paths.htmlEntries.push(resolvedPath);
-      } else {
-        paths.codeEntries.push(resolvedPath);
-      }
+      paths.codeEntries.push(resolvedPath);
+
+      // if (resolvedPath.endsWith('.html')) {
```

Comment:
could just remove this part.

Reply:
nit: should we do 

```
const codeEntries = entries.reduce(
    (paths, entry) => {
      const resolvedPath = path.resolve(inputPath, entry);

      paths.codeEntries.push(resolvedPath);
      return paths;
      },
     [] 
  );
```
?

Reply:
That's a good call out and looks a lot cleaner that way! I've gone ahead and updated this bit.